### ISSUE TITLE
ERROR - Upload handler failed to dump file .... Bad guid

### ISSUE THREAD

**Issue by gallypette** [0xrawsec/whids#7]
Hi,

I have a manager running on Ubuntu, and whids running on Win 10 (running in VBox).
I get the following type of errors popping up in the manager's output:
``` shell
2019/10/02 11:25:25 ERROR - Upload handler failed to dump file ({19480def-65c1-5d94-0000-001081270200}/c9df481b3d530a4c9d79c6cd526a186f/c9df481b3d530a4c9d79c6cd526a186f_event.json.gz): Bad guid

```

I think it's a problem of case sensitivity here:
https://github.com/0xrawsec/whids/blob/bc808541e502ad4236f113fa9c2ac348319a0db5/collector/manager.go#L41
```shell
`(?i)\{[A-F0-9]{8}-([A-F0-9]{4}-){3}[A-F0-9]{12}\}`
```
should work, but I am unsure you need case sensitivity elsewhere.

cheers,
jlouis

**Comment by qjerome** [comment]
Hi @gallypette,

Yep, that's a bug I noticed and fixed recently but not committed ...
By the way, the fix is indeed the one you suggested.

Thanks for your issue,
Cheers,

**Comment by qjerome** [comment]
I fixed the thing and made a new release v1.6.1

**Comment by gallypette** [comment]
Cool thanks!
